examples/cockroachdb/cockroachdb-petset.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
+  name: cockroachdb
+  labels:
+    app: cockroachdb
+spec:
+  ports:
+  - port: 26257
```

Comment:
please add a comment elaborating what these ports are used for (pg protocol and raft, right?)